_component_design/status-labels.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Status Labels
+parent: Notifications
+layout: component
+category: Components
+usage: >
+    Status Labels are wide tags that include short, brief, and helpful text communication regarding the progress of our users accounts.
+status: stable
+resource: true
+preview-image: preview-images/status-labels.svg
+last-modified: 2017-12-01
+helix-ui-css: true
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+## Introduction
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+Use status labels to communicate the current state of an object or process. For example, to communicate the status of a support ticket, or current health of a device.
+
+### Best Practices
+
+- To reduce visual noise when using in a table or list, label width should be fixed at the width that best accomodates the longest label with a minimum padding of 8px. 
+- Adhere to a recommended max character limit of 20 characters.
+- Use color and stroke vs fill to control how much attention a given status label commands.
```

Comment:
Also, I'm not sure what this means. Does it mean that they shouldn't fill in a label at all, but rather just change the color of the stroke to attract attention? Or that they should fill all labels and just change the color of the stroke?

Reply:
@palindromebart See Lauras comment. The sentence on line 32 seems to be unclear.

Reply:
right. let's rephrase as follows:

You can use color scheme and/or form (stroke outline vs fill) to control how much attention a given status label commands.

Reply:
Adjusted to be active voice. "- Adjust color scheme and or the form (stroke outline vs fill) of a status label to control how much attention the label commands."